server/routes/auth.js

```diff
@@ -4,13 +4,16 @@ const { validateRegister, validateLogin } = require('../validate');
 const protect = require('../middleware/auth');
 const {
   registerUser,
+  loginDemoUser,
   loginUser,
   loadUser,
   logoutUser,
 } = require('../controllers/auth');
 
 router.route('/register').post(validateRegister, registerUser);
 
+router.route('/demoLogin').post(loginDemoUser);
```

Comment:
Why are we using a separate route for 'demo-login' ? Won't it be better if we can use the existing '/login' route? :)

Reply:
right rajiv, refactoring this in the next commit ...